src/services/__mocks__/api.js

```diff
@@ -13,3 +13,7 @@ export async function fetchRestaurants() {
 export async function fetchRestaurant() {
   return {};
 }
+
+export async function postLogin() {
+  return 'ACCESS_TOKEN';
+}
```

Comment:
이렇게 될 경우, 만약, 로그인이 실패했을 경우에도 로그인이 성공한 것처럼 'ACCESS_TOKEN'이 return되는데, 어떻게 수정하는게 좋을까요?

Reply:
인자로 들어온 email과 password가 로그인 정보와 일치했을 때만 access token을 return해주는 걸로 수정해봤습니다!
<br />

```suggestion
export async function postLogin() {
  rconst EMAIL = 'tester@example.com';
  const PASSWORD = 'test';

  if (!(EMAIL === email && PASSWORD === password)) {
    return undefined;
  }

  return 'ACCESS_TOKEN';
}
```